novawallet/Modules/PayCard/PayCardInteractor.swift

```diff
@@ -38,10 +42,18 @@ final class PayCardInteractor {
 
 extension PayCardInteractor: PayCardInteractorInputProtocol {
     func setup() {
+        if let cardOpenStartTimestamp = settingsManager.novaCardOpenTimeStamp {
+            presenter?.didReceiveCardOpenTimestamp(cardOpenStartTimestamp.timeInterval)
```

Comment:
```.timestamp``` assumes value is in milliseconds. It should be TimeInterval(cardOpenStartTimestamp) I think